src/app/Http/Controllers/Auth/MyAccountController.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
```

Comment:
and i guess it is not even following psr-2 convention . And can we write use statements according to their length. Because I see laravel framework using this convention

Reply:
I guess styleci will fix it. so no problem regarding style of coding.